src/Polymorph-Widgets.package/MorphicUIManager.class/instance/lowSpaceWatcherDefaultAction..st

```diff
@@ -19,6 +19,8 @@ lowSpaceWatcherDefaultAction: theInterruptedProcess
 		newDebugSessionNamed: 'Space is low' 
 		startedAt: preemptedProcess suspendedContext.		
 	debugSession logStackToFileIfNeeded.
+	debugSession isAboutUIProcess ifTrue: [ 
+		DefaultExecutionEnvironment beActiveDuring: [self spawnNewProcess]].
```

Comment:
Hi, I did the slice export from Pharo 6 and the export using Iceberg was not working (it showed no changes to commit). So I exported the affected packages manually. It looks like sync problem with image and the working copy. I will fix the branch